etc/kubernetes/manifests/base/apps/camel-k/patches/patch-camel-k-deployment.yaml

```diff
@@ -21,6 +21,8 @@ spec:
                 name: camel-k-override-config
                 key: log.level
                 optional: true
+          - name: OPERATOR_ID
```

Comment:
maybe the above comments on line 14 and 15 can be removed?
`          #- name: KAMEL_OPERATOR_ID            
          #  value: "cos-fleetshard-operator-camel"`

Reply:
Why do we need KAMEL_OPERATOR_ID env var?

Reply:
We don't, that is why those lines can be deleted. It was an old name for the OPERATOR_ID env var

Reply:
Ah, I already deleted them! Thnaks